MAINTAINERS.md

```diff
@@ -0,0 +1,16 @@
+# Maintainers
+
```

Comment:
do we need this when we have https://github.com/project-copacetic/copacetic/blob/main/CODEOWNERS

Reply:
There's a complete maintainer list requirement and I see several other projects have both CodeOwners and Maintainers.md - thinking of updating both to be safe